### ISSUE TITLE
WAI-ARIA enhancement for toolbars

### ISSUE THREAD

**Issue by MarioBatusic** [mozilla/pdf.js#9556]
Configuration:
- Web browser and its version: ALL
- Operating system and its version: ALL
- PDF.js version: Last

Steps to reproduce the problem:
1. As a screen reader user I miss the semantic for toolbars and landmark to find a toolbar easier.
Also the keyboard navigation in a toolbar is not implemented according to WAI-ARIA spec.

What is the expected behavior? (add screenshot)
A screen reader user and other keyboard-only users should have possibility to easier skip to a toolbar, to know its accessible name and to move between buttons in a toolbar using arrow keys instead of tab.

All toolbars should be enhanced so that they can be easier located and the buttons within them easier navigated by keyboard-only users.

Example: Buttons of a toolbar should be included as follows:

<div role="region">
  <div role="toolbar" aria-label="<AnAppropriateLocalizedNameForToolbar>">
    <button aria-pressed="false/true" ...>...</button>
    <button ...>...</button>
    <button ...>...</button>
    ...   
  </div>
</div>

In case where a button toggles to show/hide the rest of the buttons of a toolbar, its state should be announced through ARIA as shown in the example of the first button above. aria-pressed MUST be set to false or true according to the current button state.

A JavaScript driven keyboard navigation should be at disposal within each toolbar to help navigate between buttons by arrow keys. For implementation see: 
https://www.w3.org/TR/wai-aria-practices-1.1/#toolbar
